backends/platform/sdl/kolibrios/wrapper-main.c

```diff
@@ -0,0 +1,56 @@
+/* ScummVM - Graphic Adventure Engine
+ *
+ * ScummVM is the legal property of its developers, whose names
+ * are too numerous to list here. Please refer to the COPYRIGHT
+ * file distributed with this source distribution.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include <kos32sys.h>
+#include <stdio.h>
+
+/* This is just a small wrapper so that the main scummvm is loaded as dll.  */
+int kolibrios_main(int argc, char *argv[]);
+
+int main(int argc, char *argv[]) {
+	printf("Loading SCUMMVM\n");
+	if (argc < 1 || strlen(argv[0]) > 3000) {
```

Comment:
Consider using `Common::strnlen()` for safer operation

Reply:
Changed to strnlen. Common:: is not available here because this file is not linked with scummvm itself until it loads it dynamically